spec/models/team_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+describe Team do
+  subject { Team.new(name: 'Patriots', city: 'New England') }
+
+  it { should validate_presence_of(:name) }
+  it { should validate_presence_of(:city) }
+  it { should validate_uniqueness_of(:name) }
+
+  it "returns a team's full name as a string" do
+    team = build(:team, name: 'Patriots', city: 'New England')
```

Comment:
Very cool, I did not know you could override attributes from an existing Factory Girl factory. This is very handy for handing expectance tests like this one.